webui/elm/public/css/fonts.css

```diff
@@ -100,6 +100,9 @@
 .icon-star::before {
   content: "\e90a";
 }
+.icon-tasks::before {
+  content: "\e920";
```

Comment:
What is this?

Reply:
pseudoelements are common for inserting text through css if that's what you're asking : )

Reply:
great question: it's a pattern typically used to display icons via font icon using a pseudo-element `::before`. Pseudo elements are reserved for visual elements only and ignored for things like accessibility (like `aria-label` and `title`)